ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -378,6 +375,40 @@ public void setupResources(final Configurable<?> config, final KsqlRestConfig ap
       config.register(lagReportingResource.get());
     }
     config.register(new ServerStateDynamicBinding(serverState));
+
+    final StatementParser statementParser = new StatementParser(ksqlEngine);
```

Comment:
nit: can we put this code into a helper method, e.g., `createWebsocketEndpoint(...)`?

Reply:
I'm going to leave it for now, as this gets moved around in a PR later in the stack